crp_APL/plan_motion_planning/include/plan_motion_planning/motionHandler.hpp

```diff
@@ -22,11 +22,15 @@ class MotionHandler : public rclcpp::Node
     MotionHandler();
 
 private:
-    void scenarioCallback(const tier4_planning_msgs::msg::Scenario::SharedPtr msg);
-    void planLatLaneFollowCallback(const autoware_planning_msgs::msg::Trajectory::SharedPtr msg);
-    void planLonEmergencyCallback(const autoware_planning_msgs::msg::Trajectory::SharedPtr msg);
-    void planLonIntelligentSpeedAdjustCallback(const autoware_planning_msgs::msg::Trajectory::SharedPtr msg);
-    void visualizeTrajectory();
+    void   scenarioCallback(const tier4_planning_msgs::msg::Scenario::SharedPtr msg);
+    void   planLatLaneFollowCallback(const autoware_planning_msgs::msg::Trajectory::SharedPtr msg);
+    void   planLonEmergencyCallback(const autoware_planning_msgs::msg::Trajectory::SharedPtr msg);
+    void   planLonIntelligentSpeedAdjustCallback(const autoware_planning_msgs::msg::Trajectory::SharedPtr msg);
+    double pointDistance(const crp::apl::Point3D & p1, const crp::apl::Point3D & p2);

```

Comment:
can we at least formally distinguish between different methods, e.g., plans and other mehotds? with comment lines or something